frontend/javascripts/admin/account/account_profile_view.tsx

```diff
@@ -63,7 +65,20 @@ function AccountProfileView() {
     },
     {
       title: "Email",
-      content: activeUser.email,
+      content: changeEmailVisible ? (
+        <ChangeEmailView onCancel={() => setChangeEmailVisible(false)} />
+      ) : (
+        activeUser.email
+      ),
+      action: (
+        <Button
+          type="default"
+          shape="circle"
+          icon={<EditOutlined />}
+          size="small"
+          onClick={() => setChangeEmailVisible(true)}
```

Comment:
What do you think about toggling `changeEmailVisible` when clicking the button? so that the view it is also rendered invisible when it was active before.

Reply:
oh okay, I saw that it was the same as for the password, but maybe its worth a thought anyway.

Reply:
Good idea. Done!